packages/bitcore-node/src/models/coin.ts

```diff
@@ -97,20 +84,80 @@ class Coin extends BaseModel<ICoin> {
       .toArray();
   }
 
-  _apiTransform(coin: Partial<MongoBound<ICoin>>, options?: { object: boolean }) {
-    let transform = {
-      _id: coin._id,
-      txid: coin.mintTxid,
-      coinbase: coin.coinbase,
-      vout: coin.mintIndex,
-      spentTxid: coin.spentTxid,
-      mintTxid: coin.mintTxid,
-      mintHeight: coin.mintHeight,
-      spentHeight: coin.spentHeight,
-      address: coin.address,
-      script: coin.script,
-      value: coin.value,
-      confirmations: coin.confirmations
+  resolveAuthhead(mintTxid: string, chain?: string, network?: string) {
+    return this.collection
+      .aggregate<{
+        chain: string;
+        network: string;
+        authbase: string;
+        identityOutputs: ICoin[];
+      }>([
+        {
+          $match: {
+            mintTxid: mintTxid.toLowerCase(),
+            mintIndex: 0,
+            ...(typeof chain === 'string' ? { chain: chain.toString() } : {}),
```

Comment:
typeof check, and then calling toString() seems redundant

Reply:
Fixing in next commit, thanks 👍